src/skins/vector/css/matrix-react-sdk/views/messages/TextualEvent.css

```diff
@@ -15,6 +15,11 @@ limitations under the License.
 */
 
 .mx_TextualEvent {
-    opacity: 0.5;
-    overflow-y: hidden;
+    font-weight: 600;
```

Comment:
Hello. We've made a few tweaks to the layout as it was being built to make it flow a little better, but this bit was as I specified in Zeplin. :)  https://zpl.io/Z1zYVaV The rhythm of the page feels a lot better when they're presented as semi-bold instead of lighter.